s3/backend.go

```diff
@@ -65,7 +65,14 @@ func (s *s3) ListBuckets() ([]gofakes3.BucketInfo, error) {
 //
 // TODO: This implementation is not ideal because it fetches all objects. We
```

Comment:
The `minio` client uses `MaxKeys` as a batch size and will thus always fetch all keys for the given path and prefix, is there a reason we are using `minio` instead of the official aws sdk?